tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -809,3 +809,107 @@ exit:
     test_driver_cipher_hooks = test_driver_cipher_hooks_init();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS */
```

Comment:
We should probably add MBEDTLS_PSA_CRYPTO_DRIVERS as a dependency as well.

Reply:
Since the test is aimed specifically at the keys exposed through the test driver, I went one step further and added a dependency on both `MBEDTLS_PSA_CRYPTO_DRIVERS` and `PSA_CRYPTO_DRIVER_TEST`.